cmd/orb.go

```diff
@@ -104,11 +137,27 @@ query ListOrbs ($after: String!) {
 
 		// Debug logging of result fields.
 		// Logger.Prettyify(result)
-
+	Orbs:
 		for i := range result.Orbs.Edges {
 			edge := result.Orbs.Edges[i]
 			currentCursor = edge.Cursor
-			Logger.Infof("Orb: %s\n", edge.Node.Name)
+			if len(edge.Node.Versions) > 0 {
+				v := edge.Node.Versions[0]
+
+				Logger.Infof("%s (%s)", edge.Node.Name, v.Version)
+
+				var orb orb
```

Comment:
`cmd/orb.go:149::warning: declaration of "orb" shadows declaration at cmd/orb.go:56 (vetshadow)`